feature/platform/transceiver/zr_input_output_power_test/README.MD

```diff
@@ -0,0 +1,112 @@
+# gNMI-1.20: Telemetry: 400ZR RX input and TX output power telemetry values streaming. 
```

Comment:
Let's assign the next test id.  You should also update testregistry.textproto by [adding a new entry right after this one](https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto#L874-L879).

```suggestion
# TRANSCEIVER-4: Telemetry: 400ZR RX input and TX output power telemetry values streaming. 
```